features/verblijfstitelhistorie.feature

```diff
@@ -1,149 +1,333 @@
+    | 60        | 24                 | 20120405            | 20090512             | 20090512               | ingangsdatum is eerder dan datum einde vorige           |
+    | 60        | 26                 | 20091129            | 20081129             | 20081129               |                                                         |
+    | 60        | 25                 |                     | 20050219             | 20050219               | datumEinde wordt ingevuld met ingangsdatum volgende     |
+    | 60        | 33                 | 20050327            | 20030327             | 20050327               | Overschrijft vorige verblijfstitel                      |
+    | 60        | 34                 | 20060327            | 20030327             | 20030327               |                                                         |
+    | 60        | 11                 | 20011205            | 20011205             | 20011205               | Ingangsdatum = datum einde                              |
+    | 60        | 29                 | 20011101            | 20010801             | 20011001               |                                                         |
+    | 60        | 30                 | 20011001            | 20010901             | 20010901               |                                                         |
 
-  Als voor een verblijfstitel ingangsdatum gelijk is aan datum einde, dan wordt deze verblijfstitel niet opgenomen in het antwoord.
+Rule: Als de peildatum waarmee geraadpleegd wordt in de geldigheidsperiode van een verblijfstitel van een persoon valt wordt die verblijfstitel opgenomen in de response.
 
-  Historische voorkomens die onjuist zijn worden niet opgenomen in het antwoord.
+  @gba
+  Scenario: Raadpleeg de verblijfstitelhistorie met peildatum na datum einde laatste verblijfstitel
+    Als de verblijfstitelhistorie wordt geraadpleegd met de volgende parameters
+    | naam                | waarde                |
+    | type                | RaadpleegMetPeildatum |
+    | burgerservicenummer | 999991553             |
+    | fields              | aanduiding, datumEinde, datumIngang |
+    | peildatum           | 2019-09-28            |
+    Dan heeft de response geen verblijfstitelhistorie
 
-  De verblijfstitels worden in het antwoord aflopend gesorteerd op ingangsdatum, zodat de meest actuele bovenaan staat.
-  Als er een actuele verblijfstitel is (categorie 10) staat deze als eerste verblijfstitel in het antwoord.
+  @gba
+  Scenario: Raadpleeg de verblijfstitelhistorie met peildatum voor de ingangsdatum van de eerste verblijfstitel
+    Als de verblijfstitelhistorie wordt geraadpleegd met de volgende parameters
+    | naam                | waarde                |
+    | type                | RaadpleegMetPeildatum |
+    | burgerservicenummer | 999991553             |
+    | fields              | aanduiding, datumEinde, datumIngang |
+    | peildatum           | 2019-09-28            |
```

Comment:
dit is dezelfde datum als vorig scenario. Dit moet een datum vóór 2001-09-01 zijn, bijvoorbeeld 2001-08-31